### PR TITLE
add members from Caicloud

### PR DESCRIPTION
**Pull request by ddysher**
@jlewi

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/74)

### CODE REVIEW

File: member_organizations.yaml [hunk 1]
```diff
@@ -25,16 +25,25 @@
...
+
 - name: Red Hat
   contacts:
     - matt@redhat.com
     - mhausenb@redhat.com
     - tmckay@redhat.com
     - willb@redhat.com
   url: https://www.redhat.com
-  
-  - name: Weaveworks
```
Comment (gaocegege) [inline member_organizations.yaml]: Here may have conflicts with #73 But LGTM

**Comment by ddysher** [comment]
@gaocegege @jlewi updated, PTAL

**Pull request merged**